i18n/en-US/articles/getting-started.md

```diff
@@ -17,7 +17,7 @@ Replace `project-name` with your project name and run the command below to start
 npx create-nullstack-app@latest project-name
 ```
 
-> 💡 If you want to start with a template using .tsx files, add `--ts` or `--typescript` to the end of command
+> 💡 If you want to start with a template using .tsx files, add `-ts` or `--typescript` to the end of command
```

Comment:
Hold on, @brunolm
The flag is really `--ts` (brought from Next and common when with more than 1 letter).
Different from that it breaks the create. There's some linter changing this?

Reply:
im changing it on create, normally shortcut flags are just a single dash